modules/Bio/EnsEMBL/Gene.pm

```diff
@@ -171,8 +170,10 @@ sub new {
   $self->external_status($external_status)
     if ( defined $external_status );
   $self->display_xref($display_xref) if ( defined $display_xref );
-  $self->biotype($type)              if ( defined $type );
```

Comment:
This counts as an interface change. In the unlikely event of users setting biotypes on gene objects, you have just broken one of the ways of doing it.

Reply:
ok, I'll put it back for now...
might be a good idea to revisit this and clean the code a bit though, I don't think it's useful.